database/migrations/2018_12_31_122052_add_network_data_indices.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddNetworkDataIndices extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('networkdata_atc', function (Blueprint $table) {
+            $table->index('callsign');
+        });
+
+        Schema::table('networkdata_pilots', function (Blueprint $table) {
+            $table->index(['departure_airport', 'arrival_airport']);
+            $table->index('departure_airport');
+            $table->index('arrival_airport');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('networkdata_atc', function (Blueprint $table) {
+            $table->dropIndex('callsign_index');
```

Comment:
This `dropIndex` needs a table prefixed as well?

Reply:
Oops! c7a74dc